lib/openstax/aws/stack.rb

```diff
@@ -220,6 +222,18 @@ def apply_change_set(params: {}, wait: false)
       change_set
     end
 
+    def revert_previous_change_set(wait: false)
```

Comment:
This reverts `to` previous changes instead of reverting previous changes, right?